src/dropdown.test.aria.ts

```diff
@@ -40,13 +40,50 @@ test.describe('filterable', () => {
         element.open = true;
       });
 
+    await page.getByRole('combobox').fill('noMatchingOptions');
+
     await expect(page.locator('glide-core-dropdown')).toMatchAriaSnapshot(`
       - text: Label
-      - combobox "Label" [expanded]
-      - listbox:
+      - combobox "Label 0 items" [expanded]
+      - text: No matching options
+    `);
+  });
+
+  test('open=${false}', async ({ page }) => {
+    await page.goto('?id=dropdown--dropdown');
+
+    await page
+      .locator('glide-core-dropdown')
+      .evaluate<void, Dropdown>((element) => {
+        element.filterable = true;
+      });
+
+    await expect(page.locator('glide-core-dropdown')).toMatchAriaSnapshot(`
+      - text: Label
+      - combobox "Label"
+    `);
+  });
+});
+
+test.describe('filter("o")', () => {
```

Comment:
Same as as in the diff above expect with a filter query to capture the input field with text in it.